datagateway_api/src/api_start_utils.py

```diff
@@ -152,7 +152,8 @@ def create_api_endpoints(flask_app, api, specs):
     # DataGateway API endpoints
     if Config.config.datagateway_api is not None:
         datagateway_api_spec = next(
-            (spec for spec in specs if spec.title == "DataGateway API"), None,
+            (spec for spec in specs if spec.title == "DataGateway API"),
```

Comment:
There's been some code formatting changes where my dev environment changes them back to the way they used to be. I assume this is because of a difference in Black versions. When running `nox -s black`, the latest version is installed. I will look into how I can (cleanly) fix a version when running that command. There's nothing for you to do here, but this serves as a reminder for me.